src/AutoRest.CSharp/Mgmt/AutoRest/MgmtOutputLibrary.cs

```diff
@@ -926,8 +929,19 @@ private TypeProvider BuildResourceData(InputType inputType, MgmtObjectType? defa
         private Dictionary<string, OperationSet> CategorizeOperationGroups()
         {
             var rawRequestPathToOperationSets = new Dictionary<string, OperationSet>();
-            foreach (var inputClient in _input.Clients)
+            foreach (var inputClient in _inputClients)
             {
+                // If the client has no operations, we skip it
+                if (inputClient.Operations.Count == 0)
```

Comment:
Isn't `_inputClient` already filtered out the client without operations?

Reply:
yes, forgot to remove these

Reply:
removed